webapp/channels/webpack.config.js

```diff
@@ -328,9 +311,7 @@ async function initializeModuleFederation() {
     }
 
     async function getRemoteContainers() {
-        const products = [
-            {name: 'playbooks'},
-        ];
+        const products = [];
```

Comment:
Thanks for leaving all of this logic. We'll see if we end up using it any time soon, but I still want to potentially use module federation for plugins in the future, so it'll be good to keep around just in case

Reply:
Sure, I definitely want to keep all the structure while we make decisions and in case we can reuse any of it